src/Microsoft.AspNetCore.Razor.Runtime/project.json

```diff
@@ -56,9 +56,12 @@
     },
     "netstandard1.3": {
       "dependencies": {
+        "System.Collections.Concurrent": "4.0.12-*",
+        "System.Reflection": "4.1.0-*",
```

Comment:
Well, there is a problem: Pulls in an inconsistent set of CoreFx packages with a newer `System.Reflection` than `System.Reflection.Extensions`, `System.Reflection.TypeExtensions` and probably other packages were built against. Question is: Is this problem worth worrying about?

(Less importantly someone could come along and waste time cleaning out the transitive dependencies.)